### ISSUE TITLE
tests are failing

### ISSUE THREAD

**Issue by mr-c** [RDFLib/html5rdf#3]
Thank you for including the tests in html5rdf 1.2.1, however they are failing.

I git bisected it to https://github.com/RDFLib/html5rdf/commit/7a9f42df260e2e445dd07d0e412c11ea7518334a 

When I run `pytest --pdb` I see failures like

```
html5rdf/tests/tokenizer.py:242: in runtest
    assert tokensMatch(expected, received, ignoreErrorOrder, True), errorMsg
E   AssertionError: 
E   
E   Initial state:
E   dataState
E   
E   Input:
E   <!DOCTYPE>
E   
E   Expected:
E   [['DOCTYPE', None, None, None, False]]
E   
E   received:
E   ['ParseError', 'ParseError', ['DOCTYPE', '', None, None, False]]
```

In addition, another error prevents pytest from showing the errors when not using `--pdb`, as see in the GitHub Actions runs here:
https://github.com/RDFLib/html5rdf/actions/runs/11587293764/job/32259253810#step:9:604

**Comment by ashleysommer** [comment]
Yes, I'm aware the tests are failing, I have seen the GitHub actions runs. I haven't had a chance to investigate it yet, other than to confirm the error does not affect the use of this library in the RDFLib project.

**Comment by mr-c** [comment]
> Yes, I'm aware the tests are failing, I have seen the GitHub actions runs. I haven't had a chance to investigate it yet, other than to confirm the error does not affect the use of this library in the RDFLib project.

Got it, thank you. For the Debian package I'll ignore the tests.
